docs/src/app/pages/LinkIt7688.md

```diff
@@ -0,0 +1,27 @@
+# Platform: Arduino 101
```

Comment:
This file is just a copy of the Arduino 101 page. Please update it before we can finish reviewing.